src/rgw/setup/rgw_service.py

```diff
@@ -40,4 +41,4 @@ def start(conf: MappedConf, config_file, log_file, motr_trace_dir, rgw_cwd, inde
             raise SetupError(e.errno, "Failed to start radosgw service. %s", e)
         except Exception as e:
             Log.error(f"Failed to start radosgw service:{e}")
-            raise SetupError(e, "Failed to start radosgw service. %s", e)
\ No newline at end of file
+            raise SetupError(e, "Failed to start radosgw service. %s", e)
```

Comment:
what is changed here. if nothing, make sure git also doesnt pick anything

Reply:
empty line was added here.
earlier it was not there hence it was showing red mark . hence fixed in this patch

Reply:
https://github.com/Seagate/cortx-rgw-integration/blob/ab608d2ee11898675929966aab67821de90a717d/src/rgw/setup/rgw_service.py

there is no empty line at the end. Please avoid any changes if it doesnt contribute to anything

Reply:
okay. sure.. @rahul27kumar could you please remove this line.